specification/source/verification.rst

```diff
@@ -556,7 +559,9 @@ as shown in :numref:`sec_ver_spe_tes_set`.
 This specifies two tests, one for the controller ``setPoiVAV`` and one for ``setPoiVAV1``.
 In this example, ``setPoiVAV`` and ``setPoiVAV1`` happen to be the same sequence, but their
 input time series and/or parameters are different, and therefore their output time series will be different.
-The test for ``setPoiVAV`` will use the globally specified tolerances, and use
+The ``generate_json`` flag will determine if the json translation for the specified model under test
+must be generated during the test using the ``modelica-json`` tool or if it is already present in the directory
+mentioned in ``modelJsonDirectory``. The test for ``setPoiVAV`` will use the globally specified tolerances, and use
```

Comment:
This is not clear:
1. In `software/verification/config_test.json` you use `generateJson` but here you used `generate_json`. Which one is correct?
2. The sentence is not clear: It states that "the flag determines whether it must be generated... or if it is already present..." Does the flag really determine whether it is already present? (Using shorter sentences is typically clearer and helps avoid ambiguities like this.)